ahnlich/ai/src/engine/ai/providers/ort.rs

```diff
@@ -252,15 +256,6 @@ impl ProviderTrait for ORTProvider {
                 storeinput_type: store_input_type,
             });
         }
-
-        let batch_size = 16;
-        let store_keys = image_inputs
-            .chunks(batch_size)
-            .try_fold(Vec::new(), |mut accumulator, batch_inputs|{
-                accumulator.extend(self.batch_inference(batch_inputs)?);
-                Ok(accumulator)
-            });
```

Comment:
I don't think we should remove the use of batches.

Reply:
If for example we have 1000 inputs, it is better to run them in batches of say 32 for example, than doing all 1000 at once.